hack/dind-cluster.sh

```diff
@@ -324,11 +324,12 @@ os::provision::install-cmds ${DEPLOYED_ROOT}"
 }
 
 function nodes-are-ready() {
-    local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
+  # Skip the SDN node since nothing is intended to be scheduled on it.
+  local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
 KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep Ready | wc -l")
-    node_count=$(echo "${node_count}" | tr -d '\r')
-    test "${node_count}" -ge "${NODE_COUNT}"
+oc get nodes | grep -v SchedulingDisabled | grep -v NotReady | grep Ready | wc -l")
```

Comment:
I have a couple of comments about this invocation:
 - why are we `docker exec`-ing into the container? Can't the local OpenShift client ping the server in the container over the network?
 - we should pass in the configuration using `oc get nodes --config=...` instead of using the `$KUBECONFIG` envar
 - Instead of the complicated multi-level `grep` that's going on here, a `jsonpath` or `go-template` structure would be better to grab the correct field, followed by a `wc -l` for counting.